prescient/engine/egret/engine.py

```diff
@@ -61,10 +62,12 @@ def initialize(self, options:Options) -> None:
         self._last_sced_pyo_solver = None
         self._data_provider = DatDataProvider()
         self._data_provider.initialize(options)
+        self._actuals_step_frequency = 60 if not options.simulate_out_of_sample \
+                                       else self._data_provider.negotiate_data_frequency(options.sced_frequency_minutes)
```

Comment:
I'm a bit confused by this conditional -- what is the relationship between `simulate_out_of_sample` and `sced_frequency_minutes`?

Reply:
`if not options.simulate_out_of_sample` means "if we're using forecasts as actuals", and forecasts are always hourly.  Only true actuals might be more frequent than once an hour.  The sced will still run at whatever frequency you've specified, but we'll be interpolating hourly forecast data to do it.